packages/assert/src/assert.js

```diff
@@ -4,297 +4,7 @@
 
 import './types';
 
-/*
-// TODO Somehow, use the `assert` exported by the SES-shim
-import { assert } from 'ses';
+import { assert } from 'ses/src/error/assert';
```

Comment:
This will not work and we do not expect to make it work without making `ses/assert` part of the public API. The reason is the opposite of what you describe.

RESM operates by lowering all modules down to CommonJS (whereas SES Compartment lifts all modules up to the level of ECMAScirpt modules).

When an RESM reaches for `ses`, it gets a version of SES that was compiled down CommonJS by Rollup. When it reaches for `ses/src/error/assert[.js]` it gets the original source, RESM compiles it down to CommonJS, Node.js attempts to load the resulting `.js` file, but since SES has `{"type": "module"}`, it is expecting that file to be in ESM format, not CJS.

It should be sufficient to import `ses` and use the `globalThis.assert` here. If we need to reach for the module from source, we will have to add `ses/assert` to the exported modules of SES and add build rules for Rollup.